docs/seq-ssi/flip-flops.md

```diff
@@ -281,6 +281,23 @@ Q(n+1) = Q(n)'J + Q(n)K'
 ````
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/47630" id="masterslave_jk_flipflop_01" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
+## Differences between SR and JK flip-flop
+
+| SR Flip-flop      |  JK Flip-flop     |
+|:------------|:--------------|
+| When J input is 1 and K input is 0 on the triggering edge of clock pulse,then output is in set state | When S input is 1 and R input is 0,output is in set state |
```

Comment:
I think the contents of SR Flip-flop and JK flip flop got reversed here.
Or you need to swap the heading of the table.

Reply:
@ayan-biswas0412, Yes sir it's by mistake, I have commit the changes. Can you please review it again...